plugins/SitesManager/templates/_trackingCodeEmail.twig

```diff
@@ -2,6 +2,26 @@
 
 {{ 'CoreAdminHome_JSTracking_CodeNoteBeforeClosingHeadEmail'|translate("'head'") }}
 
+{% if consentManagerName %}
+{{ 'CoreAdminHome_JSTracking_ConsentManagerDetected'|translate(consentManagerName, consentManagerUrl) }}
+{{ 'CoreAdminHome_JSTracking_ConsentManagerEmailNote'|translate }}
+{% endif %}
+
+{% if ga3Used %}
+{{ 'SitesManager_GADetectedEmail'|translate('Google Analytics 3', 'GA', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-3-to-matomo/')|raw }}
+{{ 'CoreAdminHome_JSTracking_ConsentManagerEmailNote'|translate }}
+{% endif %}
+
+{% if ga4Used %}
+{{ 'SitesManager_GA4DetectedEmail'|translate('Google Analytics 4', 'GA', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-4-to-matomo/')|raw }}
```

Comment:
@bx80 @sgiehl I cannot seem to find the translation for `GA4DetectedEmail` and `GTMDetectedEmail` in the [translation file](https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/SitesManager/lang/en.json) is this a bug ?

Reply:
Its a bug, have fixed it in my upcoming Pr for cloudflare